release/cathaylab.nan-latn-tw.taigi/source/cathaylab.nan-latn-tw.taigi.model.ts

```diff
@@ -0,0 +1,20 @@
+
+const source: LexicalModelSource = {
+  format: 'trie-1.0',
+  wordBreaker: {
+    use: 'default',     // we want to use the default word breaker, BUT!
+    // CUSTOMIZE THIS:
+    joinWordsAt: ['-'], // join words that contain hyphens
+  }
+  sources: ['wordlist.tsv'],
+    punctuation: {
+    insertAfterWord: ""
+  }
+  languageUsesCasing: true,
+  searchTermToKey: function(term, applyCasing) {
+    return Array.from(term)
+      .map(function(c) { return applyCasing('lower', c) };
+  }

```

Comment:
```suggestion
  searchTermToKey: function(term, applyCasing) {
    return Array.from(term)
      .map(function(c) { return applyCasing('lower', c) })
      .join('')
  }
```

When testing it this way, the predictions weren't updating due to some TS syntax issues here.  This patched that up.